### DISCUSSION TITLE
After upgrade no departments appear for HTML code

### DISCUSSION THREAD

**Discussion by alonsoac** [LiveHelperChat/livehelperchat#2174] [General]
/lhc_web/index.php/site_admin/system/htmlcodebeta

Department box just shows  Any

This happened after upgrade from 4.29 to 4.52  I see no errors in logs.

**Comment by remdex** [comment]
Missing some permissions or you have not been assigned to department

**Reply by alonsoac** [reply]
I have the check for "All departments"  and my role has permission for All modules (*)	All functions (*)
The only change I did was upgrade. On a copy that has not been upgraded it works fine.

**Reply by remdex** [reply]
Sorry, but here all works as expected. Reproduce on demo or automated hosting then I can see.
